docs/hazmat/primitives/key-derivation-functions.rst

```diff
@@ -324,6 +324,110 @@ Different KDFs are suitable for different tasks such as:
         ``key_material`` generates the same key as the ``expected_key``, and
         raises an exception if they do not match.
 
+.. currentmodule:: cryptography.hazmat.primitives.kdf.concatkdf
+
+.. class:: ConcatKDF(algorithm, length, otherinfo, backend)
+
+    .. versionadded:: 0.9
+
+    ConcatKDF (Concatenation Key Derivation Function) is defined by the NIST
+    Special Publication `NIST SP 800-56Ar2`_ document, to be used to derive
+    keys for use after a Key Exchange negotiation operation.
+
+    .. warning::
+
+        ConcatKDF should not be used for password storage.
+
+    .. doctest::
+
+        >>> import os
+        >>> from cryptography.hazmat.primitives import hashes
+        >>> from cryptography.hazmat.primitives.kdf.contcatkdf import ConcatKDF
```

Comment:
----- Original Message -----
> From: "Terry Chia" <notifications@github.com>
> To: "pyca/cryptography" <cryptography@noreply.github.com>
> Cc: "simo5" <simo@fedoraproject.org>
> Sent: Tuesday, May 5, 2015 9:32:49 AM
> Subject: Re: [cryptography] Add ConcatKDF from NIST SP 800-56A (#1898)
> 
> > +
> > +    .. versionadded:: 0.9
> > +
> > +    ConcatKDF (Concatenation Key Derivation Function) is defined by the
> > NIST
> > +    Special Publication `NIST SP 800-56Ar2`_ document, to be used to
> > derive
> > +    keys for use after a Key Exchange negotiation operation.
> > +
> > +    .. warning::
> > +
> > +        ConcatKDF should not be used for password storage.
> > +
> > +    .. doctest::
> > +
> > +        >>> import os
> > +        >>> from cryptography.hazmat.primitives import hashes
> > +        >>> from cryptography.hazmat.primitives.kdf.contcatkdf import
> > ConcatKDF
> 
> "contcatkdf" Typo here.

Fixed and pushing.

Simo.

-- 
Simo Sorce * Red Hat, Inc. * New York